packages/nimble-components/src/table-column/base/tests/table-column.stories.ts

```diff
@@ -456,3 +456,144 @@ export const sorting: StoryObj<SortingTableArgs> = {
         }
     }
 };
+
+interface ColumnWidthTableArgs extends SharedTableArgs {
+    fractionalWidth: ExampleColumnFractionalWidthType;
+    minPixelWidth: number;
+    getColumnWidthData: (
+        columnId: string,
+        args: ColumnWidthTableArgs
+    ) => number | undefined;
+}
+
+const fractionalWidthOptions = {
+    [ExampleColumnFractionalWidthType.default]: [],
+    [ExampleColumnFractionalWidthType.firstColumnHalfSize]: [
+        {
+            columnId: 'first-name-column',
+            width: 0.5
+        }
+    ],
+    [ExampleColumnFractionalWidthType.firstColumTwiceSize]: [
+        {
+            columnId: 'first-name-column',
+            width: 2
+        }
+    ],
+    [ExampleColumnFractionalWidthType.thirdColumnHalfFourthColumnTwice]: [
+        {
+            columnId: 'favorite-color-column',
+            width: 0.5
+        },
+        {
+            columnId: 'quote-column',
+            width: 2
+        }
+    ]
+} as const;
+
+const fractionalWidthDescription = `Some table columns (like \`nimble-table-column-text\`) can be configured to have a fractional width such that its width
+                                    will be proportional to other columns also configured with a fractional width. So, a column with a \`fractional-width\`
+                                    set to 2 would be twice as wide as a column with a \`fractional-width\` set to 1. The default value for
+                                    \`fractional-width\` is 1. Columns that don't support \`fractional-width\` explicitly, or another API responsible for
+                                    managing the width of the column, will behave as if they have a \`fractional-width\` of 1.`;
```

Comment:
I'm probably slow, but I had to read this several times to understand – perhaps:

```suggestion
const fractionalWidthDescription = `Configure each column's width relative to the other columns with the \`fractional-width\` property. For example, a column with a \`fractional-width\` set to 2 will be twice as wide as a column with a \`fractional-width\` set to 1. 
The default value for \`fractional-width\` is 1, and columns that don't support \`fractional-width\` explicitly, or another API responsible for managing the width of the column, will also behave as if they have a \`fractional-width\` of 1.`;
```